client/cpp/kserve-api/README.md

```diff
@@ -0,0 +1,159 @@
+                          localhost)
+      --grpc_port arg     Specify port to grpc service.  (default: 9000)
+      --timeout arg       Request timeout. (default: 0)
+```
+
+- Usage Example 
+
+```Bash
+./grpc_server_live --grpc_port 9000 --grpc_address localhost
+Server Live: True
+```
+
+### Run the Client to perform inference
+```Bash
+./grpc_infer_resnet --help
+Sends requests via KServe gRPC API.
+Usage:
+  grpc_infer_resnet [OPTION...]
+
+  -h, --help                    Show this help message and exit
+      --images_list IMAGES      Path to a file with a list of labeled 
+                                images. 
+      --labels_list LABELS      Path to a file with a list of labels. 
+      --grpc_address GRPC_ADDRESS
+                                Specify url to grpc service.  (default: 
+                                localhost)
+      --grpc_port PORT          Specify port to grpc service.  (default: 
+                                9000)
+      --input_name INPUT_NAME   Specify input tensor name.  (default: 0)
+      --output_name OUTPUT_NAME
+                                Specify input tensor name.  (default: 1463)
+      --model_name MODEL_NAME   Define model name, must be same as is in 
+                                service.  (default: resnet)
+      --model_version MODEL_VERSION
+                                Define model version.
+      --timeout TIMEOUT         Request timeout. (default: 0)
+```
+
+- Usage Example
+
+```Bash
+./grpc_infer_resnet --images_list resnet_input_images.txt --labels_list resnet_labels.txt --grpc_port 9000  
+../../../../demos/common/static/images/airliner.jpeg classified as 404 airliner 
+../../../../demos/common/static/images/arctic-fox.jpeg classified as 279 Arctic fox, white fox, Alopex lagopus 
+../../../../demos/common/static/images/bee.jpeg classified as 309 bee 
+../../../../demos/common/static/images/golden_retriever.jpeg classified as 207 golden retriever 
+../../../../demos/common/static/images/gorilla.jpeg classified as 366 gorilla, Gorilla gorilla 
+../../../../demos/common/static/images/magnetic_compass.jpeg classified as 635 magnetic compass 
+../../../../demos/common/static/images/peacock.jpeg classified as 84 peacock 
+../../../../demos/common/static/images/pelican.jpeg classified as 144 pelican 
+../../../../demos/common/static/images/snail.jpeg classified as 113 snail 
+../../../../demos/common/static/images/zebra.jpeg classified as 340 zebra 
+Accuracy 100%
+======Client Statistics======
+Completed request count 10
+Cumulative total request time 264.314 ms
+Cumulative send time 1.09484 ms
+Cumulative receive time 0.024284 ms
+```
+
+## GRPC Examples with Dummy Model
+
+This section demonstrates request to simple model, which increments each provided value. 
+
+### Start the Model Server Container with Dummy Model
+```Bash
+docker run --rm -d -v $(pwd)/src/test/dummy:/models -p 9000:9000 openvino/model_server:latest --model_name dummy --model_path /models --port 9000 
```

Comment:
this assumes user is in the root catalog and it seems it's not mentioned in this section + docs tests will probably have issues with that if at that moment we are not in a good place in the project
@ngrozae is that correct?

Reply:
That is correct. Please change path to $(pwd)/../../../src/test/dummy